latch_cli/services/cp/ldata_utils.py

```diff
@@ -52,111 +38,100 @@ class AccountInfoCurrentPayload(TypedDict):
     id: str
 
 
-class GetNodeTypePayload(TypedDict):
-    accountInfoCurrent: AccountInfoCurrentPayload
-    ldataResolvePathToNode: LdataResolvePathToNodePayload
-
-
 @dataclass(frozen=True)
 class GetNodeDataResult:
+    id: str
     name: str
     type: LDataNodeType
     is_parent: bool
 
 
 def get_node_data(
-    remote_path: str, *, allow_resolve_to_parent: bool = False
-) -> GetNodeDataResult:
-    normalized = normalize_path(remote_path)
+    *remote_paths: str, allow_resolve_to_parent: bool = False
+) -> Tuple[str, Dict[str, GetNodeDataResult]]:
+    normalized: Dict[str, str] = {}
 
-    from latch.gql._execute import execute
+    acc_sel = _parse_selection("""
+        accountInfoCurrent {
+            id
+        }
+    """)
+    assert isinstance(acc_sel, l.FieldNode)
```

Comment:
Could you move these into the _parse_selection fxn? Or have a more specific helper?

Reply:
i mean i could but i dont really see the utility here? the assert is just there for the benefit of the type checker and im fine with adding it instead of writing a `FieldNode` specific parsing helper